app/components/doi-subject.js

```diff
@@ -76,7 +76,7 @@ export default Component.extend({
   didReceiveAttrs() {
     this._super(...arguments);
 
-    if (completeSubjectList.includes(this.fragment.get('subject'))) {
+    if (completeSubjectList.includes('FOS: ' + this.fragment.get('subject'))) {
```

Comment:
i think this condition will never be `true` because the list doesn't include "FOS:" in the names. See: https://github.com/datacite/bracco/blob/e7ee5d64733c4918b88ace78f1f56ed3789d61e8/app/components/doi-subject.js#L5

which basically translates that when updating a DOI schema metadata will not be disabled if the user has preselected a FOS-subject before.

Reply:
You are right, it is the other way around, I have to strip `FOS: ` from subject. I fixed that.